File: inc.py
# author: mrozinsky (Michal Jakub Mrozinski)
# 18.09.2021 

import time
import math

start = time.time()
tmp = 0 
for i in range(1, 10000001):
	tmp = tmp + 1

end = time.time()

print("******************");
print("INC - Python Version");
print("Program that increment one variable 10000000 times (10 millions)");
print("Execution time: \n" + str(end - start)) 
print("******************");

File: inc.c
// author: mrozinsky (Michal Jakub Mrozinski)
// 18.09.2021 

#include <stdio.h>
#include <stdlib.h>
#include <time.h>
#include <stdint.h> // needed for uint32_t
#include <math.h>

int main()
{

	clock_t start = clock();
	
	uint32_t result = 0;

	for (uint32_t i=1; i<10000001; i++)
	{
		result++;
	}
		
	clock_t end = clock();
	double execution_time = (double)(end - start) / CLOCKS_PER_SEC;
	
	printf("******************\n");
	printf("INC - C Version\n");
	printf("Program that increment one variable 10000000 times (10 millions)\n");
	printf("Execution time: \n%f\n", execution_time);
	printf("******************\n");

	return 0;
	
}

